src/mumble/MainWindow.h

```diff
@@ -329,6 +329,7 @@ class MainWindow : public QMainWindow, public MessageHandler, public Ui::MainWin
 		MUMBLE_MH_ALL
 #undef MUMBLE_MH_MSG
 		void removeContextAction(const MumbleProto::ContextActionModify &msg);
+		void logChangeNotPermanent(const QString &actionName, ClientUser * const p) const;
```

Comment:
A short Doxygen comment documenting the use of this function would be nice :point_up: 
Note that we use `///` for Doxygen comments and *not* `/**`